File: 1° semestre/APC verão/Raylan da Silva Sales_lab_1/Python/lab1b.py
a = int(input("Digite um numero: "))
b = int(input("Digite outro numero: "))
c = int(input("Digite outro numero: "))
if a+b>c and a+c>b and b+c>a:
    if a==b and a==c and b==c:
        print('triangulo equilatero')
    elif a==b or a==c or b==c:
        print('triangulo isosceles')
    else:
        print('triangulo escaleno')
else:
    print('Nao eh um triangulo')

File: 1° semestre/APC verão/Raylan da Silva Sales_lab_1/C/lab1b.c
#include<stdio.h>
int main(){
	int a,b,c;
	printf("Digite um valor: ");
	scanf("%d",&a);
	printf("Digite outro valor: ");
	scanf("%d",&b);
	printf("Digite outro valor: ");
	scanf("%d",&c);
	if(((a+b)>c) && ((a+c)>b) && ((b+c)>a)){
		printf("triangulo ");
		if((a==b) && (a==c) && (b==c)){
			printf("equilatero");
		}else if ((a==b) || (a==c)|| (b==c)){
			printf("isosceles");
		}
		else if((a!=b)&&(a!=c)&&(b!=c)){
			printf("escaleno");
		}
	} else 
		printf("Nao eh um triangulo");
	return 0;
}